### PR TITLE
Copy/edit supporting documents for existing vacancies

### PR DESCRIPTION
**Pull request by cpjmcquillan**
[UD14: TEVA-440](https://dfedigital.atlassian.net/browse/TEVA-440)

**In this PR**
- Copy supporting documents from an existing vacancy (published/scheduled/expired)
- Edit supporting documents for an existing vacancy (published/scheduled/draft)
- Refactor documents controller and routing to simplify code and remove duplication. (Now passes job_id as parameter and does not use the session)
- Include notification label on review/edit page to highlight supporting documents section

**TODO**
- [x] Code review by 2+ devs
- [ ] UX review (view on staging)
- [ ] Product review (view on staging)

### CODE REVIEW

File: app/views/hiring_staff/vacancies/documents/_title.html.haml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- if @vacancy.published?
+  - content_for :page_title_prefix, "Edit the Supporting documents"
```
Comment (tatyree) [inline app/views/hiring_staff/vacancies/documents/_title.html.haml]: Shouldn't Supporting be lower case or documents be title case?

Comment (cpjmcquillan) [reply app/views/hiring_staff/vacancies/documents/_title.html.haml]: I agree. I just followed convention for all the other page titles. Happy to change them all 👍🏼

Comment (cesidio) [reply app/views/hiring_staff/vacancies/documents/_title.html.haml]: It would be nice to I18n these strings

Comment (tatyree) [reply app/views/hiring_staff/vacancies/documents/_title.html.haml]: If you're following convention then don't sweat it.  Let's get it shipped and refactor for casing/I18N later.

**Pull request merged**